Rubberduck.VBEEditor/Events/VBENativeServices.cs

```diff
@@ -188,12 +199,29 @@ private static void OnWindowFocusChange(object sender, WindowChangedEventArgs ev
 
         private static ICodePane GetCodePaneFromHwnd(IntPtr hwnd)
         {
+            if (_vbe == null)
+            {
+                return null;
+            }
+
             try
             {
                 var caption = hwnd.GetWindowText();
                 using (var panes = _vbe.CodePanes)
                 {
-                    return panes.FirstOrDefault(x => x.Window.Caption.Equals(caption));
+                    foreach (var pane in panes)
+                    {
+                        using (var window = pane.Window)
+                        {
+                            if (window.Caption.Equals(caption))
+                            {
+                                return pane;
+                            }
+                        }
+                        pane.Dispose();
```

Comment:
Don't we want to use `using` for `pane`, too? Otherwise when an exception is thrown inside the `window`'s `using` block, the `pane` will get leaked.

Reply:
The problem is that you do not want to dispose the pane returned.